tree/dataframe/inc/ROOT/RDFHelpers.hxx

```diff
@@ -15,9 +15,18 @@
 
 #include <ROOT/TypeTraits.hxx>
 #include <ROOT/RIntegerSequence.hxx>
+#include <ROOT/RDFGraphUtils.hxx>
+#include <ROOT/RDataFrame.hxx>
 
+#include <algorithm> // std::transform
 #include <functional>
 #include <type_traits>
+#include <vector>
+#include <memory>
+#include <fstream>
+#include <iostream>
+#include "TString.h"
```

Comment:
I see a lot of includes of TString: perhaps we just check if this is really necessary?